diff -Naur a/libcef/browser/file_dialog_manager.cc b/libcef/browser/file_dialog_manager.cc
--- a/libcef/browser/file_dialog_manager.cc	2026-07-26 14:47:28.899273186 +0200
+++ b/libcef/browser/file_dialog_manager.cc	2026-07-26 14:47:28.931273561 +0200
@@ -17,6 +17,7 @@
 #include "chrome/browser/file_select_helper.h"
 #include "content/public/browser/file_select_listener.h"
 #include "content/public/browser/render_frame_host.h"
+#include "content/public/browser/web_contents.h"
 #include "ui/shell_dialogs/select_file_policy.h"
 #include "ui/shell_dialogs/selected_file_info.h"
 
