diff -up chromium-146.0.7680.80/chrome/browser/first_run/first_run_internal_linux.cc.1~ chromium-146.0.7680.80/chrome/browser/first_run/first_run_internal_linux.cc
--- chromium-146.0.7680.80/chrome/browser/first_run/first_run_internal_linux.cc.1~	2026-03-13 17:54:03.000000000 +0100
+++ chromium-146.0.7680.80/chrome/browser/first_run/first_run_internal_linux.cc	2026-03-14 20:19:27.750579660 +0100
@@ -33,6 +33,8 @@ bool IsOrganicFirstRun() {
 }
 
 base::FilePath InitialPrefsPath() {
+  return base::FilePath("/etc/chromium");
+
   // The standard location of the initial prefs is next to the chrome binary.
   base::FilePath dir_exe;
   if (!base::PathService::Get(base::DIR_EXE, &dir_exe)) {
