diff -up chromium-144.0.7559.96/build/config/rust.gni.58~ chromium-144.0.7559.96/build/config/rust.gni
--- chromium-144.0.7559.96/build/config/rust.gni.58~	2026-01-17 23:59:25.000000000 +0100
+++ chromium-144.0.7559.96/build/config/rust.gni	2026-01-23 17:16:37.936072319 +0100
@@ -61,7 +61,7 @@ declare_args() {
 
   # Directory under which to find `bin/bindgen` (a `bin` directory containing
   # the bindgen exectuable).
-  rust_bindgen_root = "//third_party/rust-toolchain"
+  rust_bindgen_root = "/usr"
 
   # If you're using a Rust toolchain as specified by rust_sysroot_absolute,
   # set this to the output of `rustc -V`. Changing this string will cause all
diff -up chromium-144.0.7559.96/build/rust/rust_bindgen_generator.gni.58~ chromium-144.0.7559.96/build/rust/rust_bindgen_generator.gni
--- chromium-144.0.7559.96/build/rust/rust_bindgen_generator.gni.58~	2026-01-23 17:16:37.513736819 +0100
+++ chromium-144.0.7559.96/build/rust/rust_bindgen_generator.gni	2026-01-23 17:17:16.493960892 +0100
@@ -33,7 +33,7 @@ if (!use_chromium_rust_toolchain &&
 } else if (host_os == "win") {
   _libclang_path += "/bin"
 } else {
-  _libclang_path += "/lib"
+  _libclang_path += "/lib64"
 }
 
 # Template to build Rust/C bindings with bindgen.
