diff -up chromium-146.0.7680.164/third_party/test_fonts/fontconfig/generate_fontconfig_caches.cc.56~ chromium-146.0.7680.164/third_party/test_fonts/fontconfig/generate_fontconfig_caches.cc
--- chromium-146.0.7680.164/third_party/test_fonts/fontconfig/generate_fontconfig_caches.cc.56~	2026-03-21 00:29:56.000000000 +0100
+++ chromium-146.0.7680.164/third_party/test_fonts/fontconfig/generate_fontconfig_caches.cc	2026-03-29 00:17:53.498283967 +0100
@@ -56,7 +56,7 @@ int main() {
   FcFini();
 
   // Check existence of intended fontconfig cache file.
-  auto cache = fontconfig_caches + "/" + kCacheKey + "-le64.cache-11";
+  auto cache = fontconfig_caches + "/" + kCacheKey + "-le64.cache-8";
   bool cache_exists = access(cache.c_str(), F_OK) == 0;
   return !cache_exists;
 }
