diff -urN a/build/config/compiler/BUILD.gn b/build/config/compiler/BUILD.gn
--- a/build/config/compiler/BUILD.gn	2026-08-02 01:07:29.007033753 +0200
+++ b/build/config/compiler/BUILD.gn	2026-08-02 01:07:29.007112994 +0200
@@ -577,14 +577,15 @@
   if (is_clang) {
     # Flags for diagnostics.
     cflags += [ "-fcolor-diagnostics" ]
-    if (!is_wasm) {
-      if (!is_win) {
-        cflags += [ "-fdiagnostics-show-inlining-chain" ]
-      } else {
-        # Combine after https://github.com/llvm/llvm-project/pull/192241
-        cflags += [ "/clang:-fdiagnostics-show-inlining-chain" ]
-      }
-    }
+    # Disabled: requires newer clang than some distro toolchains provide.
+    # if (!is_wasm) {
+    #   if (!is_win) {
+    #     cflags += [ "-fdiagnostics-show-inlining-chain" ]
+    #   } else {
+    #     # Combine after https://github.com/llvm/llvm-project/pull/192241
+    #     cflags += [ "/clang:-fdiagnostics-show-inlining-chain" ]
+    #   }
+    # }
     if (diagnostics_print_source_range_info && !is_win) {
       cflags += [ "-fdiagnostics-print-source-range-info" ]
     }
