diff -up chromium-142.0.7444.59/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h.34~ chromium-142.0.7444.59/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h
--- chromium-142.0.7444.59/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h.34~	2025-10-24 18:42:30.000000000 +0200
+++ chromium-142.0.7444.59/third_party/breakpad/breakpad/src/common/linux/libcurl_wrapper.h	2025-11-05 16:34:04.461133876 +0100
@@ -35,7 +35,7 @@
 #include <string>
 #include <map>
 
-#include "third_party/curl/curl.h"
+#include <curl/curl.h>
 
 namespace google_breakpad {
 
