diff -up chromium-143.0.7499.109/third_party/blink/public/common/renderer_preferences/renderer_preferences.h.omv~ chromium-143.0.7499.109/third_party/blink/public/common/renderer_preferences/renderer_preferences.h
--- chromium-143.0.7499.109/third_party/blink/public/common/renderer_preferences/renderer_preferences.h.omv~	2025-12-15 22:34:37.441434543 +0100
+++ chromium-143.0.7499.109/third_party/blink/public/common/renderer_preferences/renderer_preferences.h	2025-12-15 22:35:18.187262959 +0100
@@ -57,7 +57,7 @@ struct BLINK_COMMON_EXPORT RendererPrefe
   bool use_overlay_scrollbar{false};
 #endif
   blink::mojom::WebRtcIpHandlingPolicy webrtc_ip_handling_policy =
-      blink::mojom::WebRtcIpHandlingPolicy::kDefault;
+      blink::mojom::WebRtcIpHandlingPolicy::kDefaultPublicAndPrivateInterfaces;
   std::vector<WebRtcIpHandlingUrlEntry> webrtc_ip_handling_urls;
   std::optional<bool> webrtc_post_quantum_key_agreement;
   uint16_t webrtc_udp_min_port{0};
