description: Disable Widevine CDM component updater
author: Michel Le Bihan <michel@lebihan.pl>

diff -urN a/chrome/browser/component_updater/registration.cc b/chrome/browser/component_updater/registration.cc
--- a/chrome/browser/component_updater/registration.cc	2026-08-02 01:06:50.130230684 +0200
+++ b/chrome/browser/component_updater/registration.cc	2026-08-02 01:06:50.130275345 +0200
@@ -137,7 +137,7 @@
 #endif  // BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
 
 #if BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
-  RegisterWidevineCdmComponent(cus);
+//  RegisterWidevineCdmComponent(cus);
 #endif  // BUILDFLAG(ENABLE_WIDEVINE_CDM_COMPONENT)
 
   RegisterSubresourceFilterComponent(cus);
